licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/pages/pdf/LicenceTaskListPage.groovy

```diff
@@ -1,7 +1,7 @@
 package uk.gov.justice.digital.hmpps.licences.pages.pdf
 
 import geb.Page
-import geb.module.Select
+import geb.module.RadioButtons
```

Comment:
these imports look like they are not being used? could they be removed?

Reply:
yep